### ISSUE TITLE
3.3.3小节关于“不定值”的描述错误

### ISSUE THREAD

**Issue by lilbedwin** [horance-liu/tensorflow-internals#16]
首先所贴的示例code，个人认为第二行多余，让人容易误解，如下：
logits = tf.matmul(y4, w5) + b5
y = tf.nn.softmax(logits)
cross_entropy = tf.nn.softmax_cross_entropy_with_logits( logits=logits, labels=t)

其次从这里的注释来看：https://github.com/tensorflow/tensorflow/blob/r1.0/tensorflow/examples/tutorials/mnist/mnist_softmax.py#L48
采用softmax_cross_entropy_with_logits，与先softmax、再手工计算交叉熵、再在batch内item间取交叉熵的均值，数学效果等同，似乎无法避免文中介绍的log(0)导致NaN问题。
